modules/pivot-table/main/index.coffee

```diff
@@ -56,7 +56,8 @@ class PivotTable extends hx.EventEmitter
       cellRender: (data, element, isHead, column) ->
         hx.select(element).text(data)
       useResponsive: true
-      data: undefined
+      data: undefined,
+      fullWidth: false
```

Comment:
Setting it to true would be a 'breaking' change - New features should be disabled by default so they don't break existing uses, something this is liable to do (i.e. causing layouts to break)